ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-extended-communities.j2

```diff
@@ -1,9 +1,9 @@
 {# eos - ip-extended-community #}
-{% if ip_extcommunity_lists is defined and ip_extcommunity_lists is not none %}
+{% if ip_extcommunity_lists is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed

So line 2 and 3 can be merged

Reply:
this check is redundant and should be removed